docs/_legacy/geth.md

```diff
@@ -85,7 +85,7 @@ IRC Freenode channels:
 * `#dappdevs`: Dapp developers channel
 * `#ethdev`: buildserver etc
 
-[IRC Logs by ZeroGox](https://zerogox.com/bot/log)
+IRC Logs by ZeroGox
```

Comment:
without the link this makes no much sense anymore - IMHO should be removed completely or a replacement should be found

Reply:
removed